UT4MasterServer/Models/Requests/WaitTimeEstimateResponse.cs

```diff
@@ -13,7 +13,7 @@ public sealed class WaitTimeEsitmate
 	[JsonPropertyName("numSamples")]
 	public int SampleCount { get; set; }
 
-	public WaitTimeEsitmate(string ratingType, double seconds, int sampleCount)
+	public WaitTimeEstimateResponse(string ratingType, double seconds, int sampleCount)
```

Comment:
I renamed class to `WaitTimeEstimateResponse`, does this sound alright?

Reply:
It sounds OK. But you should move it to Responses folder instead of Requests.